app/assets/javascripts/comment.js

```diff
@@ -64,3 +64,8 @@ function addComment(comment, submitTo, parentID = 0) {
   }
   sendFormSubmissionAjax(data, submitTo);
 }
+
+function changeicon(messageid, buttonid){
```

Comment:
The function should be camelCased and more descriptive like `changeNotificationIcon`.
`messageid` -> `messageId`
`buttonid` -> `buttonId`